content/error-handling/index.textile

```diff
@@ -0,0 +1,130 @@
+  logLevel: 1 // Errors only
+});
+```
+
+The following is an example log entry:
+
+```[text]
+LOG [Level 1]: Ably: ConnectionManager.failQueuedMessages(): failing 1 queued messages, err = [_ErrorInfo: Application Ptz0jg disabled.
+(See https://help.ably.io/error/40300 for help.); statusCode=403; code=40300]
+```
+
+h2(#meta). [meta]log channels
+
+Errors are not published in the meta channels when the client can be directly notified. For example, if a client publishes a message but exceeds a channel rate limit, the @publish()@ method uses an error callback to notify the client.
+
+There are two @metachannels@ related to logging:
+
+- @Meta[log]@ := Publishes errors that aren't visible to clients, except those related to push notifications.
+
+- @Meta[log:push]@ := Similar to @[meta]log@, but only publishes errors related to the delivery of push notifications.
+
+The following example subscribes to the @[meta]log@ channel:
+
+```[text]
+const channel = realtime.channels.get('[meta]log');
+channel.subscribe((msg) => {
+  console.log(msg);
+});
+```
+
+The following is an example event published to the @[meta]log@ channel as an @ErrorInfo@ object:
+
+```[text]
+{
+  code: 40005,
+  statusCode: 400,
+  cause: Authentication,
+  nonfatal: false,
+  href: 'https://help.ably.io/error/40005'
+}
+```
+
+h2(#console). Dev console
+
+The Dev console, accessible from your Ably dashboard, is essential for inspecting and debugging realtime errors. It visually displays all events in your application, making it particularly valuable during early-stage development or low-traffic periods when event tracking is simpler:
+
+* View all events in your application for detailed insights.
+* Test publishing and subscribing to channels to identify and resolve issues with these functions.
+
+h3(#connections). Connections
+
+Monitor state changes for example; connecting, connected, closed to identify authentication or network issues. Regularly review logs to ensure consistency and address potential problems:
+
+* Verify channels pause and resume as expected, with active and functioning subscriptions.
+* Simulate disconnection events and validate reconnection logic to ensure seamless recovery.
+* Align timestamps with reported issues to trace root causes and spot patterns.
+
+h3(#channels). Channels
+
+Address common issues such as attachment failures and undelivered messages:
+
+* Confirm the channel name and client initialization are correct. Ensure API key permissions allow the operation.
+* Verify the channel is attached and message data meets API requirements.
+* Check the @clientId@ configuration and validate the API integration using REST API commands.
+
+h2(#support). Support tickets
+
+If the provided information does not resolve your issue, contact Ably "support":https://ably.com/support. When contacting, include details such as your app ID, the error code, and any relevant logs to help troubleshoot. Include as much of the following information as possible:
```

Comment:
The second sentence here should probably be in the bullet list below instead, no?